test/extended/setup.sh

```diff
@@ -147,6 +150,9 @@ function os::test::extended::setup () {
 
 	os::log::info "Creating image streams"
 	oc create -n openshift -f "${OS_ROOT}/examples/image-streams/image-streams-centos7.json" --config="${ADMIN_KUBECONFIG}"
+
+	os::log::info "Creating sample app"
+	oc create -n openshift -f "${OS_ROOT}/examples/sample-app/application-template-stibuild.json" --config="${ADMIN_KUBECONFIG}"
```

Comment:
@jim-minter evidently this script isn't run on GCE environments.  let's chat tomorrow about how to rework this requirement, but for now i'm going to merge this since this test isn't being run on GCE today anyway.